cassandra-protocol/src/types/data_serialization_types.rs

```diff
@@ -105,20 +105,14 @@ pub fn decode_float(bytes: &[u8]) -> Result<f32, io::Error> {
 pub fn decode_inet(bytes: &[u8]) -> Result<net::IpAddr, io::Error> {
     match bytes.len() {
         // v4
-        4 => Ok(net::IpAddr::V4(net::Ipv4Addr::new(
-            bytes[0], bytes[1], bytes[2], bytes[3],
-        ))),
+        4 => {
+            let array: [u8; 4] = bytes[0..4].try_into().unwrap();
```

Comment:
Please don't use unwrap().

Reply:
This is identical to the previous code in terms of potential to panic.
`bytes[3]` is internally equivalent to `bytes.get(3).unwrap()`

Reply:
true, didn't notice that